src/mjolnir/luatagtransform.cc

```diff
@@ -126,7 +126,6 @@ Tags LuaTagTransform::Transform(OSMType type, uint64_t osmid, const Tags& maptag
       result.clear();
     }
   } catch (std::exception& e) {
-    // ..gets sent back to the main thread
```

Comment:
this was just an errant comment that i noticed while debugging. sometimes its quite annoying that we silence logs in gurka tests, especially because its the only way to see what is wrong with your lua code